### DISCUSSION TITLE
How to disable IPv6

### DISCUSSION THREAD

**Discussion by sgtcoder** [serversideup/docker-php#182] [Q&A] [answered]
I am getting an error when starting up the nginx version. There doesn't seem to be a way to disable IPv6 in the configs without cloning and modifying the docker.

```
ERROR: failed to create new listening socket: socket(): Address family not supported by protocol (97)
```

**Comment by jaydrogers** [comment, marked as answer]
This discussion came up on another project that uses our base images. Let me know if this helps! https://github.com/alexjustesen/speedtest-tracker/issues/212#issuecomment-1368031242
